.travis/systemd.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+sudo apt-get update
+sudo apt-get install -y --no-install-recommends \
+  libsystemd-dev
```

Comment:
https://github.com/DataDog/integrations-core/blob/9abdcde9528f1996ff4de35144617c1572665a18/.travis.yml#L10

Reply:
Thanks, good point, will add the lib there.